src/lib/ichinose/CAcbFile.cpp

```diff
@@ -339,9 +339,13 @@ string CAcbFile::GetSymbolicFileNameFromCueId(uint32_t cueId) {
 
 string CAcbFile::GetCueNameFromCueId(uint32_t cueId) {
     for (auto &cue : _cues) {
-        if (cue.waveformId == cueId) return string(cue.cueName);
+        if (cue.waveformId == cueId) {
+            return string(cue.cueName);
+        }
     }
-    return CAcbFile::GetSymbolicFileNameFromCueId(cueId);
+    char buffer[40] = {0};
+    sprintf(buffer, "dat_%06u.hca", cueId);
```

Comment:
The audio type is not always HCA (as you have pointed out, could be ADX or else)